src/ipc/file-system-get-entries.ts

```diff
@@ -0,0 +1,63 @@
+import makeDebug from 'debug'
+import { IpcMain, IpcMainInvokeEvent } from 'electron'
+import { promises as fs } from 'fs'
+import { isAbsolute, join } from 'path'
+import { assertHasReadAccess, HostFilePermission } from '../utils/access'
+import { Options } from '../utils/options'
+
+const debug = makeDebug('kiosk-browser:file-system-get-entries')
+
+export const channel = 'file-system-get-entries'
+
+export interface FileSystemEntry {
+  readonly name: string
+  readonly path: string
+  readonly isDirectory: boolean
+  readonly isFile: boolean
```

Comment:
do we need both? Would `isDirectory` be enough?

Reply:
in particular, I think we should not list any entires that aren't directories or files, which I guess leaves ... symlinks?

Reply:
Yeah, symlinks are one possibility. Block devices too, but we should definitely not allow access to those via e.g. `/dev`.

Reply:
Alright, we're now filtering directory entries to be only file or directory entries.